src/Umbraco.Web.UI.Client/src/views/dashboard/settings/examinemanagement.html

```diff
@@ -370,7 +370,7 @@
                                 <div class="umb-panel-group__details-status-content">
 
                                     <table class="table table-bordered table-condensed">
-                                        <caption>&nbsp;</caption>
+                                        <caption class="sr-only"><localize key="examineManagement_contentInIndex">contentInIndex</localize></caption>
```

Comment:
@BatJan, I think we could put the English translation as the `localize `element content, so that if it does not find the translation, it displays the English text instead of the key. So something like

`<localize key="examineManagement_contentInIndex">Content in index</localize>`